shopper-app/src/components/CustomTextfield.tsx

```diff
@@ -24,12 +24,31 @@ const CustomInput = React.forwardRef(function CustomInput(
   );
 });
 
+const CustomTextInput = React.forwardRef(function CustomInput(
```

Comment:
Rename to `CustomTextArea` or something else that can be easily differentiated from `CustomInput`